gradle.properties

```diff
@@ -17,11 +17,11 @@ org.gradle.parallel.threads=8
 org.gradle.parallel=true
 # Artifactory information needed.
 # ---------------------------------- InAppMessaging SDK flags ------------------------------------
-group=com.rakuten.tech.mobile
+group=com.rakuten.tech.mobile.inappmessaging
 name=inappmessaging
 description=In App Messaging Android SDK.
 #"-" indicates aar file will be uploaded to snapshot.
-version=1.4.0
+version=2.0.0
```

Comment:
Actually I think this `version` line should be completely removed. The version is set via git tags with `axion-release-plugin`. So this value is always overridden by that plugin. Having the version here as well is misleading.

Reply:
removed